agent/js/package.json

```diff
@@ -1,24 +1,45 @@
 {
-  "name": "agent-sdk",
+  "name": "agent-protocol",
   "version": "0.1.0",
   "description": "API for interacting with Agent",
+  "main": "./dist/index.js",
+  "module": "./dist/index.m.js",
+  "unpkg": "dist/index.umd.js",
+  "types": "dist/index.d.ts",
   "scripts": {
     "prepublishOnly": "pnpm build",
-    "build": "rm -rf ./dist && tsup --minify",
-    "dev": "tsup --watch",
-    "fix": "npx eslint **/src/**/*.{ts,js} --fix",
+    "clean": "rm -rf ./dist",
+    "build": "rm -rf ./dist && tsup",
+    "fix": "npx eslint \"**/{examples,src}/**/*.{ts,js}\" --fix",
     "format": "prettier --check --ignore-path .gitignore .",
     "format:fix": "prettier --write --ignore-path .gitignore ."
   },
   "private": false,
   "devDependencies": {
     "@types/express": "^4.17.17",
     "@types/node": "^20.4.1",
+    "@types/uuid": "^9.0.2",
+    "@typescript-eslint/eslint-plugin": "^5.62.0",
+    "eslint": "^8.45.0",
+    "eslint-config-standard-with-typescript": "^36.1.0",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-n": "^16.0.1",
+    "eslint-plugin-promise": "^6.1.1",
```

Comment:
They aren't used

Reply:
Add `eslint-config-prettier` to prevent the problem with `prettier`

Reply:
Done.